selfdrive/car/hyundai/carstate.py

```diff
@@ -311,11 +311,15 @@ def get_can_parser_canfd(self, CP):
       ("STEERING_SENSORS", 100),
       ("MDPS", 100),
       ("TCS", 50),
-      ("CRUISE_BUTTONS_ALT", 50),
       ("BLINKERS", 4),
       ("DOORS_SEATBELTS", 4),
     ]
 
+    if not (CP.flags & HyundaiFlags.CANFD_ALT_BUTTONS):
+      messages += [
+        ("CRUISE_BUTTONS_ALT", 50)
+      ]
+
```

Comment:
is this right?

Reply:
not super familiar with hyundai, but this matches the current setup, just without duplicate messages.

CRUISE_BUTTONS_ALT is used for the metric test, even if the buttons aren't used? https://github.com/commaai/openpilot/blob/ddddc7a8cbb9de28b24cc7d762c25b57ad43e620/selfdrive/car/hyundai/carstate.py#L169

Reply:
I think its more clear if we just add CRUISE_BUTTONS if the flag is not set